solang-parser/src/lexer.rs

```diff
@@ -33,7 +33,6 @@ pub enum Token<'input> {
 
     Struct,
     Event,
-    Error,
```

Comment:
There is also this weird `Error(string)` type, will this still work?

Reply:
I'm not sure what you mean. Are you thinking of `error Error(string);`? We don't support that yet, but the parser still supports it.